demos/src/workbox-core/updateServer.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
```

Comment:
Is it possible to put the logic in this file into a reusable `demos/src/lib/server.js` file that just gets `require()`ed and run in each individual project's `updateServer.js`?

If it's not possible to refactor things that way (I'm not sure if each Glitch project would have access to code at the `demos/src/lib/` path level), that's okay, but it would help reduce the repetition if you could.

Reply:
there's no common structure that separate glitch projects can access (as far as I'm aware). The only way would be to place every demo into the same project, each in a subdirectory. However, that might reduce clarity for users inexperienced w/ service workers.

To reduce duplication in this git repo, we could use gulp to concat the final file together.

Reply:
@jeffposnick I raised a [thread](https://support.glitch.com/t/is-there-a-way-to-access-code-in-another-glitch-project-from-another-project/22601) about this on the glitch forums. Looks like we could import this boilerplate using package.json, such as:

{
    "name": "external-package-example",
    "dependencies": {
        "backpack": "git://github.com/philkrie/demo_server.git"
    }
}

Sadly we can't target a specific directory w/in a repo, so instead of hosting this within the demos folder in the workbox repo, we might need to create a separate git repo to do this. On the plus side, if we want to expand this to all web.dev demos, we can essentially make this a package that a demo creator can import, making the git listener easier to use. What do you think?

Reply:
Eh, I'd rather not spin up a new GitHub repo if we could avoid it. I wouldn't worry about the code duplication in this case, in the interest of getting something finalized. We could always revisit after the initial setup is in place.